README.md

```diff
@@ -1 +1,139 @@
-# BooksStore
\ No newline at end of file
+# 📗 Table of Contents
+
+<h3><b>Calculator README Template</b></h3>
+
+- [📗 Table of Contents](#-table-of-contents)
+- [📖 \[Calculator\] ](#-calculator-)
+  - [🛠 Built With ](#-built-with-)
+    - [Tech Stack ](#tech-stack-)
+    - [Key Features ](#key-features-)
+  - [💻 Getting Started ](#-getting-started-)
+    - [Prerequisites](#prerequisites)
+    - [Setup](#setup)
+    - [Install](#install)
+    - [Run test](#run-test)
+    - [Run App](#run-app)
+  - [👥 Authors ](#-authors-)
+  - [🔭 Future Features ](#-future-features-)
+  - [🤝 Contributing ](#-contributing-)
+  - [⭐️ Show your support ](#️-show-your-support-)
+  - [🙏 Acknowledgments ](#-acknowledgments-)
+  - [📝 License ](#-license-)
+
+# 📖 [Calculator] <a name="about-project"></a>
+
+> Hello , This is a calculator Program where You can calculate any numbers.
+
+**[Calculator]** is a simple shelf where where You can calculate any numbers.
+
+## 🛠️ Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+
+>  HTML , CSS and React.
+
+<!-- Features -->
+
+### Key Features <a name="key-features"></a>
+
+- **[Simple Template]**
+- **[Books listed from array]**
+- **[buttons ]**
+- **[colors ]**
+- **[Dynamic styles]**
+- **[Editable texts]**
+- **[Add Delete features]**
+- **[Completed updated]**
+- **[buttons]**
+- **[colors]**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- GETTING STARTED -->
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+To get a local copy up and running, follow these steps.
+
+### Prerequisites
+
+In order to run this project you needs:
+
+A GitHub | A code editor | A web browser
+
+### Install
+
+Install all project dependencies by running the command below
+
+  `$ npm install`
+
+And run `npm start` to launch the project .
+
+### Setup
+
+Clone this repository to your desired folder:
+
+  `git clone https://github.com/hasnatali1947/Calculator/tree/setupReact`
+
```

Comment:
Great job @hasnatali1947 🤝

- [ ]  Kindly ensure your readme file references the actual project you are working on. Make sure you are referencing BooksStore instead of Calculator.